packages/fluentui/react-bindings/src/icons/createSvgIcon.tsx

```diff
@@ -26,13 +26,9 @@ export const svgIconHandledProps: (keyof SvgIconProps)[] = [
   'xSpacing',
 ];
 
-const createSvgIcon = <TProps extends SvgIconProps>({
-  svg,
-  displayName,
-  handledProps = [],
-}: SvgIconCreateFnParams<TProps>) => {
-  const Component: React.FC<React.HTMLAttributes<HTMLSpanElement> & TProps> & {
-    handledProps: (keyof TProps)[];
+const createSvgIcon = <TProps,>({ svg, displayName, handledProps = [] }: SvgIconCreateFnParams<TProps>) => {
```

Comment:
```suggestion
const createSvgIcon = <TProps>({ svg, displayName, handledProps = [] }: SvgIconCreateFnParams<TProps>) => {
```

nit

Reply:
Hm , I think it was auto formatted. Thanks!